phlib/include/phnative.h

```diff
@@ -1510,6 +1510,18 @@ PhUnloadDriver(
     _In_opt_ PCWSTR Name
     );
 
+PHLIBAPI
+NTSTATUS
+NTAPI
+PhFilterConnectCommunicationPort(
+    _In_ PPH_STRINGREF PortName,
+    _In_ ULONG Options,
+    _In_reads_bytes_opt_(SizeOfContext) PVOID ConnectionContext,
+    _In_ USHORT SizeOfContext,
+    _In_opt_ PSECURITY_ATTRIBUTES SecurityAttributes,
+    _Outptr_ HANDLE* Port
+    );
+
```

Comment:
I organized the filter APIs into their own file inside of phlib here: https://github.com/winsiderss/systeminformer/commit/38f88f9144e43f2f3aa09a361d4375b5266005d7

This should help clean up your PR a bit 👍

Reply:
1fae38674a27bfb28d20e72701c5b798e31f0d40